### PR TITLE
Add tests for Ids, Unops, and function Calls

### PR DESCRIPTION
**Pull request by akalicki**
Please remember to add tests when you add functionality to the parser! This PR adds tests for the negation `Unop`, `IDs`, and function `Calls`.

cc @lillyfwang @alexandramedway @dannyech for merge

### CODE REVIEW

File: test/parser/_id_call.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+state PI
+state myvar
+state print(40 + 2)
+state print("fourty-two")
+state myfunc(arg1, arg2)
+state prec(myvar, 2 * (2 + 3))
```
Comment (alexandramedway) [inline test/parser/_id_call.in]: Do we want to add one with no arguments?

File: test/parser/parserize.ml [hunk 2]
```diff
@@ -1,29 +1,35 @@
 open Ast 
 
-let rec eval = function
+let txt_of_op = function
```
Comment (alexandramedway) [inline test/parser/parserize.ml]: Do we need operations for anything other than binop? If not, I'm ok with removing this and simplifying the generator. If so, we definitely should keep this around.

File: test/parser/_id_call.in (hunk 1 above)
Comment (akalicki) [inline test/parser/_id_call.in]: Yeah! nice thinking

File: test/parser/parserize.ml (hunk 2 above)
Comment (akalicki) [inline test/parser/parserize.ml]: it's useful for unop as well :smiley:

**Pull request merged**